sirepo/pkcli/job_cmd.py

```diff
@@ -299,6 +308,12 @@ def _do_stateless_compute(msg, template):
     return _dispatch_compute(msg, template)
 
 
+def _error_if_response_too_large(payload):
+    if _is_over_response_size_limit(len(payload)):
+        return PKDict(state=job.COMPLETED, error="Response is too large to send")
```

Comment:
It's unclear that "completed" is useful here. Rather, return "error" and "errorCode" exactly as in the download (put) case. They could share code. I think the _dispatch_compute case might want to be able to get a file back just like download_data_file, but that's too much change right now.